pephub/routers/api/v1/project.py

```diff
@@ -93,7 +94,12 @@ async def get_namespace_projects_list(
     return agent.annotation.get_by_rp_list(registry_paths=paths, admin=namespace_access)
 
 
-@project.get("", summary="Fetch a PEP", response_model=ProjectRawRequest)
+@project.get(
+    "",
+    summary="Fetch a PEP",
+    response_model=ProjectRawRequest,
+    response_model_by_alias=False,
```

Comment:
Just curious, so I can learn... whats this for?

Reply:
pydantic has aliases, and by default, fastapi in openAPI docs is using them. I don't want to use aliases, but actual names, that's why I set it to False